scipy/stats/_binned_statistic.py

```diff
@@ -419,6 +418,10 @@ def binned_statistic_dd(sample, values, statistic='mean',
         See the `binnumber` returned value, and the `Examples` section of
         `binned_statistic_2d`.
 
+    reuse : named tuple
```

Comment:
The named tuple expects to contain names, `statistic`, `bin_edges`, and `binnumber` these are guaranteed if the `reuse` value is a `binnedStatisticddResult` but not if the namedtuple is one the user came up with on their own. Probably want to make that explicit in the docstring. The next line here underneath says `previous call to the function` in which case the input ***must*** be a `binnedStatisticddResult`.

Reply:
This is now explicit.